string input_word
---
string output_dino_name
